test/integration/commands/pluck_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Integration.Commands.PluckTest do
+  use Cog.AdapterCase, adapter: "test"
+
+  setup do
+    user = user("lucky", first_name: "Shamaus", last_name: "McLucky")
+    |> with_chat_handle_for("test")
+
+    {:ok, %{user: user}}
+  end
+
+  test "with no field option", %{user: user} do
```

Comment:
@mpeck I think trying to decouple the tests of the bare command from the templates is the way we ultimately want to go. Otherwise you need to create special "test" templates, which (by definition) only ever get used in the tests. We really shouldn't have test templates.

Having a separate little suite of tests to ensure that all our templates work, given some pre-defined input, would be a good thing to have, too.